packages/cdk-ecs-alb/lib/index.ts

```diff
@@ -77,13 +71,6 @@ export class CdkEcsAlb extends cdk.Construct {
             ],
             entryPoint: ["powershell", "-Command"],
             command: ["C:\\ServiceMonitor.exe w3svc"],
-            logging: ecs.LogDriver.awsLogs({
```

Comment:
Why was the log driver removed?

Reply:
Matt correct me if I'm wrong but I think he's removing it because it makes it impossible to update the stack without completely deleted. So if we just wanted to update the task def for example we can't because the log group prevents us from updating.

Reply:
@traviswalker26  The log driver was removed because it exists in the task definition which is being imported by arn.  Once again, there's sucky cdk support around importing a task definition into an ecs service in cdk https://github.com/aws/aws-cdk/issues/6240#issuecomment-635907319.  A viable workaround for the time being has been found: Create the ecs service with a temp task definition and then use an escape hatch to override the property in the cfn template to hold the arn of the actual imported task definition that you want.  I tested this with the current webwin-ecs-posapi stack and it works well.